data/map systems.txt

```diff
@@ -6428,6 +6554,45 @@ system Aya'k'k
 		distance 3009.66
 		period 2983.58
 
+system "Barnard's Star"
```

Comment:
```suggestion
system Barnard
```

Having a _system_ (rather than the star itself) named "Barnard's Star" seems a bit awkward. We could just name it Barnard, in honor of E. E. Barnard.

Reply:
I think it's kind of jarring the way you have it. I know the star is called "Barnard's Star" in real life, but we're naming a star system. Including the word "star" in the name just feels redundant.

Reply:
Maybe we should add more `<person>'s Star` systems...

I nominate Plot's Star, in honour of the inventor of the hyperdrive.

Reply:
I'd have to agree that it is a bit jarring. If we did it with other systems, I think it would be fine as "----'s Star" is more than a few examples irl, but if it's just the one, it would appear to be a special case compared to other systems when it shouldn't be. Either changing it to Barnard, or having a different name entirely would be preferable, if we don't have other systems with the -'s Star part.

Reply:
Another name for Barnard's Star is "Proxima Ophiuchi", but then you lose the name recognition of Barnard's Star.

Reply:
Ah, I didn't see the other "-'s Star" name, so I think it's workable. On that note, since we're adding more Pirate systems eventually, we'll have to have names for those systems too, so perhaps that's another area that we could have more "-'s Star" names.

Reply:
The other "-'s Star" name is also being added in this PR, though. Vanilla currently has nothing like that.